Account/FirefoxAccountConfiguration.swift

```diff
@@ -157,28 +235,46 @@ public struct ChinaEditionFirefoxAccountConfiguration: FirefoxAccountConfigurati
     public let pushConfiguration: PushConfiguration = FirefoxPushConfiguration()
 }
 
-public struct ChinaEditionSync15Configuration: Sync15Configuration {
-    public init() {
-    }
+public protocol Sync15Configuration {
+    init(prefs: Prefs?)
 
-    public let tokenServerEndpointURL = URL(string: "https://sync.firefox.com.cn/token/1.0/sync/1.5")!
+    var tokenServerEndpointURL: URL { get }
 }
 
-public protocol Sync15Configuration {
-    init()
-    var tokenServerEndpointURL: URL { get }
+public struct ChinaEditionSync15Configuration: Sync15Configuration {
+    public init(prefs: Prefs? = nil) {
```

Comment:
This change isn't needed if you remove the constructor from the `Sync15Configuration`.